cogboard-webapp/src/components/PopoverWithControls.js

```diff
@@ -39,7 +46,20 @@ export const PopoverWithControls = ({
       >
         {withCopy ? <Button onClick={copyBody}>Copy</Button> : null}
         <Button onClick={handlePopoverClose}>Close</Button>
-        <StyledPopoverText>{body}</StyledPopoverText>
+        {bodyMessage === 'NO MATCH' ? (
+          <StyledPopoverTextWrapper>
+            <StyledPopoverText>
+              <StyledPopoverHeader>EXPECTED RESPONSE BODY</StyledPopoverHeader>
```

Comment:
just `EXPECTED RESPONSE` and `ACTUAL RESPONSE`

Reply:
Yeah and please move this to const too

Reply:
Fixed in commit 297f448